package.json

```diff
@@ -38,6 +38,7 @@
   },
   "homepage": "https://github.com/w3c/vc-di-ecdsa-test-suite#readme",
   "dependencies": {
+    "@digitalbazaar/credentials-v2-context": "github:digitalbazaar/credentials-v2-context",
```

Comment:
direct from github OK?

Reply:
it doesn't have a release yet, there are some oddities to this release, we have an existing credentials-context so I think we want to figure out a way of a single credentials-context having multiple versioned contexts inside of it, but no one has really figured that out yet.